tgui/packages/tgui/interfaces/Vote.tsx

```diff
@@ -225,10 +225,10 @@ const ListPolls = (_, context) => {
       <Section title="Начать голосование">
         <Stack vertical justify="space-between">
           {polls ? (
-            polls.map((poll) => (
+            polls.map(poll => (!poll.adminOnly || !!isAdmin) && (
```

Comment:
```suggestion
            polls.map(poll => (!poll.adminOnly || isAdmin) && (
```
конвертация в булеан сделана в бекенде, тут уже не надо получается